biomage/rotate_ci/rotate_ci.py

```diff
@@ -225,6 +225,18 @@ def rollback_if_necessary(iam, keys, result_codes):
 
     return success
 
+def exclude_iac_from_rotation(repos, org_name):
+    warning_message = """
+        This script WILL NOT rotate any secrets in the iac repository.
+        If you want to rotate secrets in the iac repository, you will need to do that either manually,
+        or by adjusting this script to fit the environment structure of the secrets in iac.
+    """
+    click.echo(click.style("WARNING: " + warning_message, fg="yellow"))
```

Comment:
printing a warning message that iac will be excluded from rotation, just in case someone forgets and thinks that the secrets there got rotated